docs/content/latest/admin/yb-tserver.md

```diff
@@ -25,53 +25,283 @@ $ ./bin/yb-tserver \
+
+#### --max_clock_skew_usec
+
+Specifies the expected maximum clock skew between any two nodes in your deployment.
+
+Default: `50000` (50ms)
+
+#### --rpc_bind_addresses
+
+Specifies the comma-separated list of addresses to bind to for RPC connections.
+
+Default: `0.0.0.0:9100`
+
+#### --server_broadcast_addresses
+
+Public IP or DNS hostname of the server (along with an optional port).
+
+Default: `0.0.0.0:9100`
+
+#### --use_private_ip
+
+Determines when to use private IP addresses. Possible values are `never` (default),`zone`,`cloud` and `region`. Based on the values of the [placement (`--placement_*`) configuration options](#placement-options).
+
+Default: `never`
+
+#### --webserver_interface
+
+Address to bind for the web server user interface.
+
+Default: `0.0.0.0`
+
+#### --webserver_port
+
+The port for monitoring the web server.
+
+Default: `7000`
+
+#### --webserver_doc_root
+
+Monitoring web server home.
+
+Default: The `www` directory in the YugabyteDB home directory.
+
+### Logging options
+
+#### --logtostderr
+
+Flag to log to standard error (`stderr`).
+
+#### --log_dir
+
+Specifies the directory to store `yb-tserver` log files.
+
+Default: Same as [`--fs_data_dirs`](#fs-data-dirs)
+
+### Placement options
+
+#### --placement_zone
+
+Specifies the name of the availability zone, or rack, where this instance is deployed.
+
+Default: `rack1`
+
+#### --placement_region
+
+Specifies the name of the region, or data center, where this instance is deployed.
+
+Default: `datacenter1`
+
+#### --placement_cloud
+
+Specifies the name of the cloud where this instance is deployed.
+
+Default: `cloud1`
+
+### YSQL options
+
+The following options, or flags, support the use of the [YSQL API](../../api/ysql/).
+
+#### --enable_ysql
+
+Enables the YSQL API. Replaces the deprecated `--start_pgsql_proxy` option.
+
+Default: `false`
+
+#### --ysql_enable_auth
+
+Enables YSQL authentication.
+
+Default: `false`
+
+#### --pgsql_proxy_bind_address
+
+Specifies the bind address for the YSQL API.
+
+Default: `0.0.0.0:5433`
+
+#### --pgsql_proxy_webserver_port
+
+Specifies the web server port for YSQL metrics monitoring.
+
+Default: `13000`
+
+#### --ysql_hba_conf
+
+Specify a comma-separated list of YugabyteDB setting assignments.
+
+Default: `""`
+
+#### --ysql_pg_conf
+
+Comma-separated list of PostgreSQL setting assignments.
+
+Default: `""`
+
+#### --ysql_timezone
+
+Specifies the time zone for displaying and interpreting timestamps.
+
+Default: `""` — Uses the YSQL time zone.
+
+#### --ysql_datestyle
+
+Specifies the display format for data and time values.
+
+Default: `""` — Uses the YSQL display format.
+
+#### --ysql_max_connections
+
+Specifies the maximum number of concurrent YSQL connections.
+
+Default: `0`
+
+#### --ysql_default_transaction_isolation
+
+Specifies the default transaction isolation level.
+
+Default: `""`
```

Comment:
```suggestion
Default: `"repeatable read"`
```

Reply:
Changed to :

Default: `READ COMMITTED` (implemented in YugabyteDB as `REPEATABLE READ`)